apps/rush-lib/assets/rush-init/common/config/rush/command-line.json

```diff
@@ -118,7 +118,12 @@
        *
        * For details, refer to the website article "Using watch mode".
        */
-      "watchForChanges": false
+      "watchForChanges": false,
+
+      /**
+       * Disable cache for this action. This may be useful if this command affects state outside of projects' own folders.
```

Comment:
Clarify that it disables the "cloud build cache".  Otherwise it's kind of unclear which cache you are referring to, since "cache" is pretty vague. Rush has lots of caches.

Maybe the setting name should be `disableBuildCache` or `disableCloudBuildCache`.